rbac/management/group/view.py

```diff
@@ -430,6 +433,7 @@ def roles(self, request, uuid=None):
         @apiHeader {String} token User authorization token
 
         @apiParam (Path) {String} id Group unique identifier.
+        @apiParam (Query) {String} order_by Determine ordering of returned roles.
```

Comment:
Should we add this for the principals? And a test similar to the one you added for roles

Reply:
Oh! Yes! I definitely need to update the docs. I wasn't thinking of adding a test for the principals, since it would essentially just be directly returning whatever I throw in the mock data, but I'll look back at it and see what I can put together.

Reply:
I think that if we can enumerate the valid values for order_by in the openapi doc that would be great. Also, will there be a parameter to control ascending/descending order?

Reply:
I'll work on adding them into the docs, @catastrophe-brandon. Thanks for pointing that out! I'm working on allowing both ascending and descending order and trusting in the BOP to manage the conversation with IT for us